Assets/HoloToolkit/Utilities/Scripts/Editor/ProjectSettingsWindow.cs

```diff
@@ -16,7 +16,7 @@ namespace HoloToolkit.Unity
     public class ProjectSettingsWindow : AutoConfigureWindow<ProjectSettingsWindow.ProjectSetting>
     {
         private const string SharingServiceURL = "https://raw.githubusercontent.com/Microsoft/MixedRealityToolkit-Unity/master/External/HoloToolkit/Sharing/Server/SharingService.exe";
-        private const string InputManagerAssetURL = "https://raw.githubusercontent.com/Microsoft/MixedRealityToolkit-Unity/master/ProjectSettings/InputManager.asset";
+        private const string InputManagerAssetURL = "https://raw.githubusercontent.com/Microsoft/MixedRealityToolkit-Unity/Dev_Unity_2017.2.0/ProjectSettings/InputManager.asset";
```

Comment:
Should this change to master now? Pending my PR though, I guess.

Reply:
Yup